src/dataloader/dataloader.py

```diff
@@ -130,16 +136,25 @@ def __init__(self, config, database_name="APTOS"):
         self.train_batch_size = config["train_batch_size"]
         self.test_batch_size = config["test_batch_size"]
         self.valid_batch_size = config["valid_batch_size"]
-        self.total_image_num = config["num_images"]
+        '''
+        based on issue #19, minimum total images are set to 1000. if config["num_images"] > 1000,
```

Comment:
I added a comment in the issue that this minimum value should be much smaller.

Reply:
Added, if you think it looks good, can you resolve the comment?

Reply:
The comments are unedited.

Reply:
My bad, added that. Can you resolve the comments and merge?